webui/src/components/RepositoryCreateForm.js

```diff
@@ -18,20 +18,33 @@ export const RepositoryCreateForm = connect(
     };
   })(({ error, onSubmit, onCancel, create, sm = 6, config }) => {
     const fieldNameOffset = 3;
+    const repoValidityRegex = /^[a-z0-9][a-z0-9-]{2,62}$/;
+    const storageNamespaceValidityRegex = /^(s3|gs|mem|local|transient):\/.*$/;
 
     const [formValid, setFormValid] = useState(false);
+    const [repoValid, setRepoValid] = useState(true);
```

Comment:
This should start as false since it's empty, right?

Reply:
Ideally yes. But if it starts as false then the error messages will be displayed the first time the modal opens. And as we have initialized `formValid` as false, it will anyways prevent the user from submitting the form without any input.

Reply:
Gotcha, that will do.